docker/common/docker-run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Copyright (c) The Libra Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+set -e
+
+if [ "${COMMAND_TO_RUN}" = "key-manager" ]; then # Execute the key manager
```

Comment:
I'd rather use the docker command to select what to run. Also this needs to be backwards compatible for now. So:
* Remove this script
* Make `/safety-rules.sh` the CMD in the dockerfile
* Have `run-key-manager.sh` specify `/key-manager.sh` as the command